integration/dockerfiles/Dockerfile_test_copy_symlink

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.11 as t
+RUN apk add gcc
+
+FROM scratch
+COPY --from=t /usr/lib/libstdc++.so.6 /usr/lib/
```

Comment:
is /usr/lib/libstdc++.so.6 a symlink in alpine?

Reply:
Yes it is, see the original issue:
```
lrwxrwxrwx    1 root     root            19 Jan 13 19:11 /usr/lib/libstdc++.so.6 -> libstdc++.so.6.0.27
```

In my opinion a test like this would suffice and be less prone to errors (libstdc++.so changing version) than the current approach:
```Dockerfile
FROM alpine
FROM scratch
COPY --from=0 /bin/sh /bin/sh
```
`/bin/sh` is a symlink to `/bin/busybox` on Alpine and is one of the core principles.

Reply:
yes @samos123  it is a symlink in alpine image which is causing an error.
Thanks @wilmardo.